src/main.rs

```diff
@@ -265,16 +265,27 @@ async fn forward_request(
     fn handle_result(
         result: Result<(HttpResponseAny,), AgentError>,
     ) -> Result<HttpResponseAny, Result<Response<Body>, Box<dyn Error>>> {
+        // https://internetcomputer.org/docs/current/references/ic-interface-spec#reject-codes
+        const DESTINATION_INVALID: u64 = 3;
```

Comment:
Could we maybe maintain a little enum at the top of the file with relevant codes?